grails-app/controllers/org/pih/warehouse/api/PutawayApiController.groovy

```diff
@@ -52,6 +52,7 @@ class PutawayApiController {
             putawayItem.inventoryLevel = InventoryLevel.findByProductAndInventory(putawayItem.product, putaway.origin.inventory)
             putawayItem.quantityAvailable = productAvailabilityService.getQuantityOnHandInBinLocation(putawayItem.inventoryItem, putawayItem.currentLocation)
         }
+        response.setHeader("Cache-Control", "no-store")
```

Comment:
why was this needed?

Reply:
Oh yea, you cced Artur twice :smile: 
![image](https://github.com/openboxes/openboxes/assets/83239466/386162a0-21f3-4d94-9c10-159d655f96a9)
Now, I got it, thanks

Reply:
:see_no_evil:

Reply:
I am still baffled by the need for this

Reply:
I feel like it is something that we need to discuss and decide.
I wanted to show the issue and a potential solution.

If we feel like disabling the cache in the header is not the way to go, just a single validation should be enough to prevent users mess with old data

Reply:
Yeah, I feel like, having the validation should be enough. However, you can showcase or explain the cache store  tomorrow morning